client/jupyterlab/jupyterlab_easeml/src/index.tsx

```diff
@@ -0,0 +1,230 @@
+import {ILayoutRestorer, JupyterFrontEnd, JupyterFrontEndPlugin, ILabShell} from '@jupyterlab/application';
+import {ICommandPalette, MainAreaWidget, WidgetTracker} from '@jupyterlab/apputils';
+import {Message} from '@phosphor/messaging';
+import {Widget} from '@phosphor/widgets';
+import Vue from 'vue';
+import BootstrapVue from 'bootstrap-vue';
+import EASEML_VUE from './Iframe.vue'
+import EASEML_SIDEBAR_VUE from './Sidebar.vue'
+import * as React from "react";
```

Comment:
Is it possible to avoid the dependency on React and do it all in Vue.js?

Reply:
fixed